Us3 page d accueil

Kuroshin-28 opened the pull request.

Kuroshin-28 committed:
Massive changes :
- Redo the whole carousel mechanic and add hammer.js in case of touch
  event going wrong.
- Reduce website description text
- Modify CSS for both carousel and description

Kuroshin-28 committed:
Small variable name change on service (Some shitty french over there)

Kuroshin-28 committed:
Merge branch 'develop' into 'US3-Page_d_Accueil'

bastienapp commented:
J'ai ajouté tes collègues, je validerai après relecture de l'en d'entre eux :) (merci de m'ajouter comme reviewer à ce moment-là)

Kuroshin-28 committed:
Update hammer-config.ts

Kuroshin-28 committed:
Update home.component.html

Matthieu28 reviewed (commented):
Un commentaire a enlever dans ton hammer-config.ts

Kuroshin-28 committed:
Minor changes :
- Image CSS corrected for better render
- Minor changes like commentaries suppression and type attribution for
  function

Kuroshin-28 committed:
Merge branch 'US3-Page_d_Accueil' into 'US3-Page_d_Accueil'

Kuroshin-28 committed:
Change src route for carousel slider-element
